docs/plugins/filters/standard/isFlaggedUser/README.md

```diff
@@ -0,0 +1,22 @@
+--8<-- "docs/plugins/filters/standard/isFlaggedUser/reference.md"
```

Comment:
I think the having `standard` in the path is redundent, from few reaons
1. it might confuse users when they take it into their repos and keep this in the path
2. it doesnt allow to make sure we dont have duplicates in the plugins directory, if we allow plugins to be added under `plugins/filters` for all kind of plugins then GitHub file system will make sure plugins are unique by name

Reply:
Fixed